en/Import notes/Import Textbundle files.md

```diff
@@ -0,0 +1,25 @@
+---
+permalink: import/textbundle
+---
+Obsidian allows you to easily migrate your notes from apps that support the [Textbundle](https://textbundle.org/) format using the [[Importer|Importer plugin]].
+
+The Textbundle format bundles Markdown text and all referenced images into a single file, providing a more seamless way to move out of a sandboxed application. Textbundle export is supported by [many apps](https://textbundle.org/), including Agenda, Craft, Taio, Ulysses, Zettlr, and more.
+
+## Import Textbundle files into Obsidian
+
+You will need the official Obsidian [[Importer]] plugin, which you can [install here](obsidian://show-plugin?id=obsidian-importer).
+
+1. Open **Settings**.
+2. Go to **Community Plugins** and [install Importer](obsidian://show-plugin?id=obsidian-importer).
+3. Enable the Importer plugin.
+4. Open the **Importer** plugin using the command palette or ribbon icon.
+5. Under **File format** choose **Textbundle (.textbundle, .textpack).**
+6. Select the location of your Textbundle or Textpack files. Zip files are also supported.
+7. Click **Import** and wait until import is complete.
+8. You're done!
+
+## Troubleshooting
+
+### No files are selected for import
+
+The textbundle import can only import one textbundle file at a time, and only on Mac devices. Instead, you may compress all of the textbundle files you wish to import into a single .zip file and import this instead from any device.
```

Comment:
@kepano and @tgrosinger What do you think of:

On MacOS devices, the textbundle import will import one textbundle file at a time. To process multiple textbundle files at once, you will need to compress all of the textbundle files into a single `.zip` file before importing with Importer.

Reply:
If we determine it happens on all Apple devices, we can adjust MacOS accordingly.

Reply:
🤔 A slight change then. I am not sure how 

On MacOS devices, Importer will process only one textbundle file per import. To process multiple textbundle files at once, you will need to compress all of the textbundle files into a single .zip file before importing with Importer.

Alternatively....

On MacOS devices, Importer will process only one textbundle file per import. Multiple individual textbundle files will require individual import attempts. 

To more efficiently process multiple textbundle files at once, you will need to compress all of the textbundle files into a single .zip file before importing with Importer.

Reply:
Instead of "will process" can we say "can only select"?

Reply:
I'm fine with that. :)